content/en/docs/security/config-best-practices.md

```diff
@@ -115,10 +115,129 @@ addresses so the network functions properly in dual-stack environments and
 applications, where both protocol versions are used.
 
 If you are working in environments that have nonstandard networking setups, such
-as Docker or Kubernetes, see the
-[example configurations](https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/security-best-practices.md#safeguards-against-denial-of-service-attacks)
-in our component developer documentation for ideas on how to bind your component
-endpoints.
+as Docker or Kubernetes, `localhost` might not work as expected. The following
+examples show setups for the OTLP receiver gRPC endpoint. Other Collector
+components might need similar configuration.
+
+#### Docker
+
+You can run the Collector in Docker by binding to the correct address. Here is a
+`config.yaml` configuration file for an OTLP exporter in Docker:
+
+```yaml
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: my-hostname:4317 # Use the same hostname from your docker run command
+```
+
+In your `docker run` command, use the `--hostname` argument to bind the
+Collector to the `my-hostname` address. You can access the Collector from
+outside that Docker network (for example, on a regular program running on the
+host) by connecting to `127.0.0.1:4567`. Here is an example `docker run`
+command:
+
+```shell
+docker run --hostname my-hostname --name container-name -p 127.0.0.1:4567:4317 otel/opentelemetry-collector:0.104.0
```

Comment:
@mx-psi Should we update this command with the most recent Collector version? Same question for lines 155 and 196.

Reply:
Yes, that makes sense to me, we can always point to the latest version

Reply:
you should be able to pick up the version with a shortcode, I don't have it in my head right now, but maybe it's used in some of the other collector pages already and you can copy it here

Reply:
Thanks, I found the short code.